src/components/AttributeProject.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { memo } from 'react';
+
+export type AttributeProjectProps = {
+  project: IProject;
+};
+
+const AttributeProject = ({ project }: AttributeProjectProps) => {
+  if (!project) {
+    return null;
+  }
+  return (
+    <aha-flex>
+      <a href={project.url} target="_blank">
```

Comment:
can we use a sanitizing wrapper like we've done in other places to prevent XSS if project.url is e.g. `javascript:alert(1)` ? https://github.com/aha-develop/azure-devops-repos/blob/a88f3d00e48ea042e036593197d0d965f918e2c0/src/components/ExternalLink.tsx 
https://github.com/aha-develop/github/blob/68ab1a3e512b9e6d70c3d30bf2342b52a1678ca4/src/components/ExternalLink.js

@trydionel I wonder if we should put that component in https://github.com/aha-develop/aha-develop-react for easier re-use

Reply:
@cornu-ammonis Yea, agreed. We're seeing that pattern in just about every component.

Reply:
Opened https://big.aha.io/develop/features/ADT-118